iceoryx2-bb/elementary/BUILD.bazel

```diff
@@ -22,6 +22,7 @@ filegroup(
 rust_library(
     name = "iceoryx2-bb-elementary",
     srcs = glob(["src/**/*.rs"]),
+    crate_features = [ "testing" ],
```

Comment:
Okay, I think here we have a problem. I totally overlooked that this one needs to be part of `rust_test_suite`. But in that won't work and if it is part of `rust_library` all consumers of `iceoryx2-bb-elementary` will have access to the `BumpAllocator`.

It seems what we want to achieve is not compatible with bazel :/

Reply:
probably we can define 2 rust_library for different purposes, one for users and one for testing. I will try it today to see whether it works or not.

Reply:
@elBoberido never mind, I have defined a new `rust_library` inside elementary with testing feature and define an alias inside `container` for the test cases. So it doesn't affect the common `rust_library` definitions.